src/args.rs

```diff
@@ -20,41 +20,83 @@ use crate::{
     ArgGroup::new("query_or_filter")
         .multiple(true)
         .required(true)
-        .args(&["path_to_query_file", "query_source", "filter"])
+        .args(&["path_to_query_file", "query_text", "filter"])
 ))]
 pub struct Args {
     paths: Vec<PathBuf>,
 
-    #[arg(short = 'Q', long = "query-file", conflicts_with = "query_source")]
+    /// The path to a tree-sitter query file.
+    ///
+    /// This conflicts with the --query-text option.
+    #[arg(short = 'Q', long = "query-file", conflicts_with = "query_text")]
     pub path_to_query_file: Option<PathBuf>,
 
+    /// The source text of a tree-sitter query.
+    ///
+    /// This conflicts with the --query-file option.
     #[arg(short, long, conflicts_with = "path_to_query_file")]
-    pub query_source: Option<String>,
-
+    pub query_text: Option<String>,
+
+    /// The name of the tree-sitter query capture
+    /// (without leading "@")
+    /// whose matching nodes will be output.
+    ///
+    /// By default this is the "first" capture encountered
+    /// in the query source text.
     #[arg(short, long = "capture")]
     pub capture_name: Option<String>,
 
+    /// The target language for matching.
+    ///
+    /// By default all files corresponding to supported
+    /// languages will be searched if the provided query
+    /// can be successfully parsed for that language.
```

Comment:
Is rustfmt doing the wrapping here? (It is doing it with a narrower width than I'd expect.)

Reply:
Ah I guess I didn't know it would do that for me, updated to "let it do the line-breaking"